board/grinn/liteboard-4g/rootfs/scripts/power_telit.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+gpioset `gpiofind "GSM_ONn"`=1
+sleep 2
+gpioset `gpiofind "GSM_ONn"`=0
+
```

Comment:
Remove last empty line. Also try to unify that file for both chiliboard and liteboard. Maybe something like that:
`gpioset -m time -s 2 `gpiofind "GSM_ONn"`=1
gpioset `gpiofind "GSM_ONn"`=0`
Or maybe it is a good idea to first assing a variable with gpiofind command and then use that variable.